CONTRIBUTING.rst

```diff
@@ -68,3 +68,22 @@ Coding guidelines
 Nilearn follows the coding conventions used by scikit-learn. `Please read them
 <http://scikit-learn.org/stable/developers/contributing.html#coding-guidelines>`_
 before you start implementing your changes.
+
+Documentation
+--------------
+
+To build our documentation, we are using `sphinx <https://www.sphinx-doc.org/en/master/usage/quickstart.html>`_ for the main documentation and `sphinx-gallery <https://sphinx-gallery.github.io/stable/index.html>`_ for the example tutorials.
+If you want to make changes to the example tutorials, please do the following :
+
+1. First, ensure that you have installed sphynx and sphynx gallery
+2. Clone the nilearn repostiory and go to ``nilearn/examples``
+3. Make your changes using `reStructuredText files <http://docutils.sourceforge.net/rst.html>`_
+4. You can now go to `nilearn/doc` and build the examples locally:
```

Comment:
Do the rst codeblock: starts with "::" and indented after.

Reply:
do you have a working example ? I tried few times but it does not seem to work
::
>>my code 
and also
::>>my code